test/lifecycle/error.test.ts

```diff
@@ -272,4 +272,84 @@ describe('error', () => {
 			somePretty: 'json'
 		})
 	})
+
+	describe('handle scoped error', async () => {
```

Comment:
@SaltyAom If an exception is thrown, how should be the behavior? Should be the same as calling `error`, or should it go directly to the global error handler (i.e., `onError`)?

Currently, is the latter: it goes directly to the `onError`.
BUT I think that the expected behavior is going to the local handler first.